guides/searching-items.md

```diff
@@ -79,7 +79,7 @@ Code Version
 : `code_version:abcdef` finds items that have been seen in the code_version `abcdef`.
 
 Item Number
-: `#123` finds the item with counter number `123`
+: `#123` finds the item with counter number `123`. You can search for multiple items if you separate each item by two spaces, e.g. `#123  #456`.
```

Comment:
Hm it just worked on prod for me on the RB account....would you mind trying it again @brianr ? someone wrote in about it and I thought it would be a useful thing to add.